layers/land.yaml

```diff
@@ -9,25 +9,9 @@ layers:
                 color: global.boundary_color
                 width: 1px
 
-    waterareas:
-        data: { source: jawg , layer: waterarea }
-        filter: { not: { tunnel: 1 }}
-        draw:
-            polygons:
-                order: 4
-                color: global.water_color
-
-        shore_lines:
-            filter: { $zoom: { min: 16 } }
-            draw:
-                lines-coast:
-                    order: 5
-                    width: [[16,20px],[18,10m]] # actually visually 1/3
-                    color: global.water_color
-
     water:
-        data: { source: jawg }
-        filter: { not: { tunnel: 1 }}
+        data: { source: jawg, layer: water }
+        filter: { not: { structure: tunnel }}

```

Comment:
`structure` is not a field of `water`

Reply:
Oh but what about waterways that go through tunnels, like culverts? Are they left out from the data set?

Reply:
Nope, they are included, there is no difference between culverts and non culverts waterways.
From overpass with jawg background (streets-v1):
![data](https://user-images.githubusercontent.com/5153882/95222672-078f1200-07f9-11eb-8335-4acf14fb9f87.png)
![raw map](https://user-images.githubusercontent.com/5153882/95222781-24c3e080-07f9-11eb-83c0-444cf495cb13.png)

Reply:
Mh okay, then I leave the filter as it is, in case this gets added some time in the future, can't hurt to have it in the style now.